src/ring_theory/power_series.lean

```diff
@@ -888,24 +1071,23 @@ end trunc
 
 end comm_semiring
 
-section comm_ring
-variables [comm_ring α]
-
-instance : comm_ring (power_series α) := by delta power_series; apply_instance
+section ring
+variables [ring α]
 
 instance C.is_ring_hom : is_ring_hom (C : α → power_series α) :=
 mv_power_series.C.is_ring_hom
 
+instance coeff_zero.is_ring_hom :
```

Comment:
Now that bundled `ring_hom`s are here, maybe we should be using that.

Reply:
Done.